book.rb

```diff
@@ -0,0 +1,12 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+  end
+
```

Comment:
- [ ] It would be great if you make the following changes as suggested by the previous reviewer 👍

1. First kindly initialize the rentals variable in your initialize method 😊
2. It would be great if you define your rentals variable in attr_accessor 👍
3. Please add the add_rental method as suggested by the previous reviewer 🤔

After making these changes your book class would look like below ⬇️ This is just a snippet of the sample of code 👍 Kindly don't copy paste it because it is just a structure 🤔

```ruby

class Book

  attr_accessor :title, :author, :rentals

  def initialize(title, author)
    .
    .
    @rentals = []
  end  

  def add_rental(person, date)
    Rental.new(date, self, person)
  end
end
```